web/src/beta/lib/core/engines/Cesium/Feature/Resource/utils.ts

```diff
@@ -120,7 +121,7 @@ export const attachStyle = (
   entity: Entity,
   layer: ComputedLayer | undefined,
   evalFeature: EvalFeature,
-  currentTime: JulianDate,
+  viewer: Viewer,
```

Comment:
Just curious, why do we want the WHOLE viewer? Isn't that kinda heavy and it seems like you still are just using it for the currentTime 🤔

Reply:
Fixed! https://github.com/reearth/reearth/pull/813/commits/2bc17cfff30b64258ac1b35738d70ac3f21d723f